defaults/main.yml

```diff
@@ -3,22 +3,44 @@
 ---
 # defaults file for ansible-coreos-kubelet
 coreos_kubelet_systemd: /etc/systemd/system
+etc_kubernetes: /etc/kubernetes
 coreos_kubelet_install_dir: /opt/bin
-kubelet_version: "v1.7.0_coreos.0"
-kubelet_aci: "quay.io/coreos/hyperkube"
+var_lib_kubelet: /var/lib/kubelet
+
+gcr_repo: "gcr.io/google_containers"
```

Comment:
similar feedback as the other change. Better to call this variable something non-specific to `gcr`

Reply:
It is a gcr repo :) I am removing it since we have just one repo.